pkg/webhook/cspc.go

```diff
@@ -488,6 +489,13 @@ func (wh *webhook) validateCSPCUpdateRequest(req *v1beta1.AdmissionRequest, getC
 		return response
 	}
 	pOps := NewPoolOperations(wh.kubeClient, wh.clientset).WithNewCSPC(&cspcNew).WithOldCSPC(cspcOld)
+
+	if ok, msg := pOps.ValidateScaledown(); !ok {
+		err = errors.Errorf("invalid cspc specification: %s", msg)
```

Comment:
Good to log after this line so that it will be helpful for debugging

Reply:
As this error will be displayed on the console when scaledown validation fails and the patch or update done by the user will not be executed so no need for log here like other cases.

Reply:
It will help in debugging time.

Reply:
As the operation was not successful (the current state of cspc did not change) and the user can see the error in front of him how will logging the same error help?

Reply:
Done